google/cloud/bigquerycontrol/v2/internal/job_rest_stub.cc

```diff
@@ -104,9 +104,18 @@ DefaultJobServiceRestStub::ListJobs(
                    rest_internal::DetermineApiVersion("v2", options), "/",
                    "projects", "/", request.project_id(), "/", "jobs"),
       rest_internal::TrimEmptyQueryParameters(
-          {std::make_pair("all_users", request.all_users() ? "1" : "0"),
+          {std::make_pair("all_users", (request.all_users() ? "1" : "0")),
+           std::make_pair("max_results",
```

Comment:
FYI (mostly for myself): I was expecting that we would need to say `max_results.value` here. But from experimenting, both `max_results` and `max_results.value` work.

```sh
$ curl -X GET -H "Authorization: Bearer $(gcloud auth print-access-token)" \
  https://bigquery.googleapis.com/bigquery/v2/projects/cloud-cpp-testing-resources/jobs?maxResults=1
# response with 1 result

curl -X GET -H "Authorization: Bearer $(gcloud auth print-access-token)" \
  https://bigquery.googleapis.com/bigquery/v2/projects/cloud-cpp-testing-resources/jobs?maxResults.value=1
# response with 1 result
```

Strange... :shrug: